src/locale/sv.yml

```diff
@@ -1211,6 +1211,18 @@ feat:
       personRecordColumn: Svarande
       suggestedPeople: Föreslagna personer
       unlink: Avlänka
+    surveyForm:
+      accept: Jag accepterar nedanstående villkor
+      policy:
+        link: https://zetkin.org/sv/sekretess
+        text: Klicka här för att läsa Zetkins sekretesspolicy
+      sign:
+        anonymous: Signera anonymt
+        nameAndEmail: Signera med namn och e-mail
+      submit: Skicka
+      terms:
+        description: När du skickar in enkäten kommer informationen att lagras i Zetkin och hanteras av {organization} i deras verksamhet, i enlighet med Zetkins sekretesspolicy.
+        title: Sekretesspolicy
```

Comment:
This would actually not normally be updated by the programmer. The YAML files are automatically generated (from the interface at translate.zetkin.org where translators work) and should never be written to directly by a human. It's not a big deal though.